template/ios/HelloWorld/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+  <string>>$(MARKETING_VERSION)</string>
```

Comment:
Looks like an extra `>`

Reply:
cc @cipolleschi ☝️ careful when importing

Reply:
Should we change Display Name automaticcaly following xcode settings? with change to $(PRODUCT_NAME) ?

Reply:
I wonder how's possible that the CI is green if the `info.plist` is malformed... :/

Reply:
> I wonder how's possible that the CI is green if the `info.plist` is malformed... :/
> 
> > Should we change Display Name automaticcaly following xcode settings? with change to $(PRODUCT_NAME) ?
> 
> Probably yes. It's better to have a single source of truth and to follow the platform's standard.

is our PR is malformed? on my side working fine. please check again sir.
i have test changed `CFBundleDisplayName `to use `$(PRODUT_NAME)` when installing on emulator&real device somehow its still use project name,not following settings on XCODE display name.

Reply:
@erfinbadrian you can see that there is one extra `>` in the change here `  <string>>$(MARKETING_VERSION)`

Reply:
> @erfinbadrian you can see that there is one extra `>` in the change here ` <string>>$(MARKETING_VERSION)`

ahh my bad.
i dont see that sorry. I fixed that. Thank you!